lambdas/authorisation/src/main/java/uk/gov/di/data/lep/Authorisation.java

```diff
@@ -2,17 +2,54 @@
 
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.auth0.jwt.JWT;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import software.amazon.lambda.powertools.logging.Logging;
 import software.amazon.lambda.powertools.tracing.Tracing;
 
-public class Authorisation implements RequestHandler<Object, Object> {
+import java.util.Collections;
+import java.util.HashMap;
+
+public class Authorisation implements RequestHandler<APIGatewayProxyRequestEvent, AuthoriserResponse> {
+    protected static Logger logger = LogManager.getLogger();
 
     @Override
     @Tracing
     @Logging(clearState = true)
-    public Object handleRequest(Object input, Context context) {
-        var logger = context.getLogger();
-        logger.log("Authenticating and authorising request");
-        return null;
+    public AuthoriserResponse handleRequest(APIGatewayProxyRequestEvent event, Context context) {
+        logger.info("Authenticating and authorising request. Event parameter: " + event.toString());
```

Comment:
Could you add a comment saying this is placeholder logic, and then also remove the logger that could log PII